schema.sql

```diff
@@ -0,0 +1,23 @@
+CREATE TABLE surveys(
+   id serial PRIMARY KEY,
+   name text NOT NULL
+ );
+                                                        
+ CREATE TABLE questions(
+  id serial PRIMARY KEY,
+  survey_id int REFERENCES surveys(id) ON DELETE CASCADE NOT NULL,
+  text text NOT NULL
+);
+
+ CREATE TABLE contributors(
+   id serial PRIMARY KEY,
+   email_address text UNIQUE NOT NULL
+ );
+
+ CREATE TABLE responses(
+   id serial PRIMARY KEY,
+   question_id int REFERENCES questions(id) NOT NULL,
+   contributor_id int REFERENCES contributors(id) NOT NULL,
+   text text NOT NULL,
```

Comment:
Same here, again, I know you were aware of this, but while I personally can't think of a better name, I do think 'responses.response' is more clear.